stdlib/public/Concurrency/TaskGroup.swift

```diff
@@ -16,8 +16,11 @@ import Swift
 
 /// Starts a new scope that can contain a dynamic number of child tasks.
 ///
-/// A group waits for all of its child tasks
-/// to complete or be canceled before it returns.
```

Comment:
This was the _only_ place our of like 8 places where we still had the wrong sentence here. 

Specifically the "or to be cancelled" is wrong and was already fixed in other places but somehow we missed this one place in the docs so fixing it now